server/lib/manifest.py

```diff
@@ -89,13 +90,33 @@ def add_tale_creator(self):
                                         user=self.user,
                                         force=True)
         self.manifest['createdBy'] = {
-            "@id": self.tale['authors'],
+            "@id": tale_user['email'],
             "@type": "schema:Person",
             "schema:givenName": tale_user.get('firstName', ''),
             "schema:familyName": tale_user.get('lastName', ''),
             "schema:email": tale_user.get('email', '')
         }
 
+    def add_tale_authors(self):
+
+        self.manifest['schema:author'] = []
+        for author in self.tale['authors']:
+            self.manifest['schema:author'].append(self.create_author_record(author))
+
+    def create_author_record(self, author):
+        """
+        Creates a record for an author that is associated with a Tale
+        :param author: An object that describes the author
+        :return: A dictionary listing of the author
+        """
+        return {
+            "@id": author['orcid'],
```

Comment:
We need to fail if orcid isn't present.  This will immediately happen for every existing Tale, since we don't have orcids for any user.  At the beginning of manifest creation, check for the orcid and return an error if not present "You must provide and ORCID for this user..."